src/KaliClubSig.sol

```diff
@@ -277,6 +283,8 @@ contract KaliClubSig is ClubNFT, Multicall, IClub {
             }
         }
 
+        if (!success) revert ExecuteError();
```

Comment:
would this cause multi-sig to freeze if a transaction can't execute, since txs are in order of nonce?

Reply:
No, as long as we increment the nonce after this, which should be done anyway to prevent any sort of reentrancy bug. We should also consider a reentrancy guard in general for this function.